src/features/environmentDetails/components/EnvironmentDetails.tsx

```diff
@@ -68,6 +68,8 @@ export const EnvironmentDetails = ({
   );
   const [artifactType, setArtifactType] = useState<string[]>([]);
   const [showDialog, setShowDialog] = useState(false);
+  const [defaultEnvIsChanged, setDefaultEnvIsChanged] = useState(false);
```

Comment:
```suggestion
  const [defaultEnvVersIsChanged, setDefaultEnvVersIsChanged] = useState(false);
```

This is becoming a super long variable, but for now I'd rather err on the side of being explicit with vars and what they do/are